netsim/augment/links.py

```diff
@@ -568,8 +568,13 @@ def set_interface_address(intf: Box, af: str, pfx: netaddr.IPNetwork, node_id: i
 """
 def IPAM_unnumbered(link: Box, af: str, pfx: typing.Optional[bool], ndict: Box) -> None:
   for intf in link.interfaces:
-    if not af in intf:            # No static address, set it to link bool value or use loopback AF presence for old-style unnumbereds
-      intf[af] = pfx if isinstance(pfx,bool) else bool(ndict[intf.node].get(f'loopback.{af}',False))
+    if not af in intf:                            # No static address
+      if isinstance(pfx,bool):                    # Set it to link bool value if it exists
+        intf[af] = pfx
```

Comment:
Shouldn't this be set only if ```True```?

Reply:
The 'false' values are scrubbed later in the process.

This also allows someone overly-creative to define (for example) `unnumbered: True, ipv6: False`. I know this is contrived, but I've seen worse 🤷‍♂️